lib/rails_cloud_tasks/scheduler.rb

```diff
@@ -55,6 +55,8 @@ def build_job(job)
     def parse_jobs_from_file
       settings = File.read(File.expand_path(scheduler_file_path))
       YAML.safe_load(ERB.new(settings).result).map(&:deep_symbolize_keys)
+    rescue Errno::ENOENT
+      []
```

Comment:
If possible, you could split this in two cases (i) file not found and (ii) invalid format and log those errors (info and error, respectively).